projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/ExecutionDataFetcher.java

```diff
@@ -305,4 +286,52 @@ private static DataJobPage buildResponse(
             .totalPages(pageSize)
             .totalItems(count).build();
    }
+
+   List<V2DataJob> populateStatusCounts(List<V2DataJob> dataJos, DataFetchingEnvironment dataFetchingEnvironment) {
+
+      var statusesToCount = determineStatusesToCount(dataFetchingEnvironment);
```

Comment:
Good article for `var` usage - https://www.lewuathe.com/when-to-use-var-in-java-10.html. Personally I don't like `var` and I don't think all variables should be declared via `var`.

Reply:
There are some good points in this article and I agree with it, and I must admit I am too reliant on IntelliJ telling me the type of var. However I am sure there are a lot of similar usages of var in our codebase and we have nothing in our coding standard that says how to use it. In fact we have nothing in our coding standard about java code. - https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard . We should pick something and be consistent with it.